MTA-4505 Release Notes and Attributes Update for 7.2.2 (#135)
migtools/mta-documentation: docs/topics/ref_resolved-issues-7-2-2.adoc

// Module included in the following assemblies:
//
//docs/release_notes-7.2.0/master.adoc

:_template-generated: 2024-12-04
:_mod-docs-content-type: REFERENCE

[id="resolved-issues-7-2-2_{context}"]
= Resolved issues

This section provides highlighted issues that have been resolved in {ProductFullName} 7.2.2.

.MTA CLI completes analysis for compiled Java binary without dependencies
Previously, when analyzing applications that did not include dependencies in the compiled Java binary, {ProductShortName} CLI failed the analysis because no dependency was found. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-3415[(MTA-3415)]

.MTA CLI allows the use of `--target` flag for custom targets 
Previously, to create a custom target, {ProductShortName} CLI did not allow the `--target` flag in analysis. The analysis did not complete even though custom rules were available for the custom target. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-4629[(MTA-4629)]

> .MTA CLI analyses generate consistent effort for applications with private repository source control

Reviewer: ```suggestion
.MTA CLI analysis generates consistent effort for applications with private repository source control
```

> Previously, in the {ProductShortName} CLI, analyses of an application with a private repository source control generated varying effort. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-2984[(MTA-2984)] 

Reviewer: ```suggestion
Previously, in the {ProductShortName} CLI, analysis of an application with a private repository source control generated varying effort. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-2984[(MTA-2984)] 
```

> Previously, in the {ProductShortName} CLI, analyses of an application with a private repository source control, generated varying effort. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-2984[(MTA-2984)] 

Reviewer 2: Hi @Pkylas007, I guess after the phrase "private repository source control" there is an unnecessary comma

Author: Thanks, I removed the comma.

.Issue report suggests fix for issues after an analysis in the MTA CLI

> Previously, the HTML issue report did not describe the fix for issues, after completing an analysis in the {ProductShortName} CLI. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-4782[(MTA-4782)]

Reviewer 2: here too after  _fix for issues,_

> .Weblogic application analysis in MTA CLI generates more issues

Reviewer: ```suggestion
.WebLogic application analysis in MTA CLI generates more issues
```

Author: Thank you, applied the change.

> Previously, the Weblogic rule did not trigger issues for the `PromoService` class under the proprietary logger category, after analyzing a Java application in the {ProductShortName} CLI. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-3845[(MTA-3845)]

Reviewer: ```suggestion
Previously, the Oracle WebLogic Server rule did not trigger issues for the `PromoService` class under the proprietary logger category, after analyzing a Java application in the {ProductShortName} CLI. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-3845[(MTA-3845)]
```

Author: Thank you, updated the text.

> .Metric target statuses for `RHSSO` operator are `up` for Tackle addon

Reviewer: ```suggestion
.Metric target statuses for `RHSSO` operator are `up` for Tackle add-on
```

Author: Thank you, made the change.

> Previously, when you configured `tackle` with authentication for the {ProductShortName} operator, the statuses of `RHSSO` operator metric targets were `down` in the {ocp-short} console. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-4816[(MTA-4816)]

Reviewer: ```suggestion
Previously, when you configured `tackle` with authentication for the {ProductShortName} operator, the statuses of Red Hat Single Sign On (`RHSSO`) operator metric targets were `down` in the {ocp-short} console. This issue has been resolved in {ProductShortName} 7.2.2. link:https://issues.redhat.com/browse/MTA-4816[(MTA-4816)]
```

Author: Thanks, updated the content.
